simulation/sub8_gazebo/nodes/gazebo_controller.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, target='sub8::base_link'):
 
         self.reset_srv = rospy.Service('gazebo/reset_gazebo', ResetGazebo, self.reset)
         self.state_pub = rospy.Publisher('odom', Odometry, queue_size=1)
-        self.world_state_pub = rospy.Publisher('world_odom', Odometry, queue_size=1)
+        self.map_state_pub = rospy.Publisher('map_odom', Odometry, queue_size=1)
```

Comment:
I think this should be world. Map frame is zeroed at start, but the world frame is constant in gazebo (i.e. zero is the center of the world and not where the sub starts)

So it is a world frame not the map frame